api/cloudprotocol/certificates.go

```diff
@@ -64,10 +64,13 @@ type RenewCertData struct {
 
 // UnitSecret keeps unit secret used to decode secure device password.
 type UnitSecret struct {
-	Version int `json:"version"`
-	Data    struct {
-		OwnerPassword string `json:"ownerPassword"`
-	} `json:"data"`
+	OwnerPassword string `json:"ownerPassword"`
+}
+
+// UnitSecrets keeps secrets for nodes.
+type UnitSecrets struct {
+	Version string                `json:"version"`
+	Nodes   map[string]UnitSecret `json:"nodes"`
```

Comment:
Should be map[string]string I guess. Please check https://github.com/aosedge/aos_protocols/blob/main/src/cloud_common/protocols/unit/certificates.py

Reply:
fixed